composer.json

```diff
@@ -1,8 +1,6 @@
 {
     "name": "johansatge/workflowy-php",
     "description": "An unofficial WorkFlowy API",
-    "version": "0.2.3",
```

Comment:
I didn't use composer/Packagist for a long time, isn't the version mandatory?

Reply:
Nope, you do tags, and packagist gets your info through that, knowing which version it is.